### ISSUE TITLE
The number of ammunition and items in the gear section of the character sheet cannot be edited

### ISSUE THREAD

**Issue by theczarek** [Muttley/foundryvtt-ac2d20#136]
When I edit these fields, they revert to their previous values. 

![image](https://github.com/user-attachments/assets/539780a6-5db4-4803-b271-e4693dc55271)

**Comment by Muttley** [comment]
Yes, looks like a bug.  You can get round the problem by editing the actual item directly (right-click -> edit button)
